helm/templates/guacamole/postgres.volume.yaml

```diff
@@ -6,7 +6,9 @@
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
-  name: {{ .Release.Name }}-volume-guacamole-postgres
+  name: {{ .Release.Name }}-guacamole-postgres
```

Comment:
Good point, but couldn't we do something like explained here: https://stackoverflow.com/questions/48403542/how-to-rename-persistence-volume-claim ? I think that it would be good to have a clear and consistent naming scheme for our k8s objects also making it easier to work with k8s/helm.

Reply:
Of cause, the migration can be done manually. To automate it, we would need a script to backup the data, run `helm upgrade` and restore the backup in the PVC.

I'm ok if you create an issue for it.